ticketutil/servicenow.py

```diff
@@ -0,0 +1,459 @@
+            if isinstance(user, str):
+                user = [user]
+
+            fields = {'watch_list' : ', '.join(user)}
+            params = self._create_ticket_parameters(fields)
+            self.s.headers.update(self.headers_post_)
+            r = self.s.put(self.ticket_rest_url, data=params)
+            r.raise_for_status()
+            self.ticket_content = r.json()['result']
+            logging.info('CC list rewritten for {0}'
+                         .format(self.ticket_id))
+        except requests.RequestException as e:
+            logging.error('Failed to rewrite CC list')
+            return False
+
+    def remove_cc(self, user):
+        """
+        Removes user(s) from cc list.
+        :param user: A string representing one user's email address, or a list
+        of strings for multiple users.
+        :return:
+        """
+        try:
+            logging.info('Removing user(s) from CC list')
+            watch_list = self.ticket_content['watch_list'].split(',')
+            watch_list = [item.strip() for item in watch_list]
+            if isinstance(user, str):
+                user = [user]
+            for item in user:
+                if item in watch_list:
+                    watch_list.remove(item)
+
+            fields = {'watch_list' : ', '.join(watch_list)}
+            params = self._create_ticket_parameters(fields)
+            self.s.headers.update(self.headers_post_)
+            r = self.s.put(self.ticket_rest_url, data=params)
+            r.raise_for_status()
+            self.ticket_content = r.json()['result']
+            logging.info('User(s) removed from CC list of {0}'
+                         .format(self.ticket_id))
+        except requests.RequestException as e:
+            logging.error('Failed to remove user(s) from CC list')
+            return False
+
+    def _prepare_ticket_fields(self, fields):
+        """
+        Makes sure each key value pair in the fields dictionary is in
+        the correct form.
+        :param fields: Ticket fields.
+        :return: fields: Ticket fields for the ticketing tool.
+        """
+        for key, value in fields.items():
+            if key in ['opened_for', 'operating_system', 'category', 'item',
+                        'severity', 'hostname_affected', 'opened_by_dept']:
+                fields['u_{}'.format(key)] = value
+                fields.pop(key)
+            if key == 'topic':
+                fields['u_topic_reportable'] = value
+                fields.pop(key)
+            if key == 'email_from':
+                fields['u_email_from_address'] = value
+                fields.pop(key)
+        return fields
+
+
+    def devops_one_url(self):
+        """
+        Creates DevOps One URL of the existing ticket
+        """
+        return '{server}/pnt/?id=ticket&sys_id={sys_id}&table={table}'.format(
+                server=self.url, sys_id=self.sys_id, table=self.project)
+
+
+def test():
```

Comment:
It looks like test() and test_comment() are functional tests that you run to test out your code? I think these should be removed for now, and placed in a separate file that we don't commit to the ticketutil repo. I have files like this for all the other ticketutil tools, but I don't push them up to ticketutil because they contain Red Hat internal URLs and wouldn't apply to anyone else using the code.

Or we can keep them, but place them in a separate tests/ directory, similar to this: https://github.com/jstavel/ticketutil/blob/30498c0e2b683872b4bb75422f48c5dde4e77359/tests/test_bugzilla_attachments.py

I would like to roll out a functional test suite across all ticketutil tools at some point, but haven't started on this yet.